src/components/database-replica.ts

```diff
@@ -65,6 +65,7 @@ const defaults = {
   maxAllocatedStorage: 100,
   instanceClass: 'db.t4g.micro',
   enableMonitoring: false,
+  engineVersion: '15.5',
```

Comment:
Lint: `engineVersion` is missing from the `DatabaseReplicaArgs` type.

Reply:
Should we set up a CircleCI job for this repo? It can just run linter for start, but we can (should) add tests later on.

Reply:
@mbareta We could also add publishing to `npm` to the ci/cd eventually which would streamline that part.